### PR TITLE
[SKIT-352]: Update Dapp schema to add verification details

### PR DESCRIPTION
**Pull request by sayantan-b**
**Notes For Reviewers**

### Final Checklists

- [X] Yes, I have added the ticket number in my PR description like `[SKIT-11] Desc`
- [X] Yes, I haven't made any secret variables public all are listed in .env file
- [X] Yes, I have reviewed my PR once by myself
- [X] Yes, I have tested my PR in my local environment and everything is working as expected

[SKIT-11]: https://btpck.atlassian.net/browse/SKIT-11?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/interfaces/dAppSchema.ts [hunk 1]
```diff
@@ -181,4 +181,5 @@ export interface DAppSchema {
   ownerAddress?: string;
   tokenId?: string;
   metrics?: object;
+  verification?: object;
```
Comment (r4881t) [inline src/interfaces/dAppSchema.ts:184]: Just `object` is not sufficient. You need to have the full schema of verification also available.

**Review by r4881t — CHANGES_REQUESTED** [review]
Any changes to the schema json file?

File: src/interfaces/dAppSchema.ts (hunk 1 above)
Comment (sayantan-b) [reply src/interfaces/dAppSchema.ts]: Yes I have added the Verification Schema.

**Review by rishikesh-y — APPROVED** [review]
LGTM

**Review by r4881t — APPROVED** [review]

**Pull request merged**